RadixWallet/Features/P2PLinksFeature/Row/P2PLinkRow+Reducer.swift

```diff
@@ -4,8 +4,8 @@ import SwiftUI
 // MARK: - P2PLink
 public struct P2PLinkRow: Sendable, FeatureReducer {
 	public struct State: Sendable, Hashable, Identifiable {
-		public typealias ID = ConnectionPassword
-		public var id: ID { link.connectionPassword }
+		public typealias ID = P2PLink
```

Comment:
We may still want to use connectionPasword or the public key, instead of using the Hash value of P2PLink. We may add later the functionality allowing users to change the name of P2PLink, in which case the ID will change dynamically.

Reply:
The ID will not change. We have a custom implementation of `Equatable` and the ID is basically the `publicKey`
```swift
extension P2PLink: Equatable {
	public static func == (lhs: Self, rhs: Self) -> Bool {
		lhs.id == rhs.id
	}
}
```

Reply:
Oh, but only `Equatable`, not `Hashable`